src/labels/CustomLabels.labels

```diff
@@ -723,6 +723,14 @@
         <shortDescription>Context for when to add a date, amount, or detail field to a Rollup__mdt record.</shortDescription>
         <value>These fields determine what information is being summarized and provide the normal defaults for Donations. You can edit them as needed.</value>
     </labels>
+    <labels>
+        <fullName>CRLP_BatchCalculating</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Displays on page when "Recalculate Rollups" button is hit on Acc/Con/GAU/RD.</shortDescription>
+        <value>The rollups batch job is calculating and may take a few moments to complete. Go back to the {0} and refresh to see updated rollup values.</value>
```

Comment:
@bethbrains do we want @smarovich to review this message?

Reply:
She did! This is her reviewed message. :)